client/src/pages/sign-in.tsx

```diff
@@ -15,7 +15,7 @@ export default function SignIn() {
   const router = useRouter();
 
   const handleLogin = async (formData: FormData) => {
-    const username = formData.account;
+    const email = formData.account;
     const password = formData.password;
```

Comment:
you can remove these variables since they are only used in the `axios.post` request